libs/shared-types/src/lib/TpCompanyProfile.ts

```diff
@@ -22,6 +22,9 @@ export type TpCompanyProfile = {
 
   state: TpCompanyProfileState
 
+  howDidHearAboutRedi?: string
```

Comment:
If we go for the key-value object approach, can we suffix this with something like `id` or `key`? I don't have a strong opinion about which one is better. I guess `key` is nice and generic.